modules/custom/openy_popups/openy_popups.info.yml

```diff
@@ -3,3 +3,6 @@ type: module
 description: Implements OpenY Popups.
 core: 8.x
 package: OpenY
+version: '8.x-1.0'
+dependencies:
+  - openy_session_instance
```

Comment:
why popups depends from session instance?
Looks like something to improve

Reply:
@hamrant I suggest adjusting `openy_popups` module and make a soft dependency. In the case of the module `openy_session_instance not installed just do not display the link `Change` on camps and branches. Add a condition to check is module installed or not and if yes use `Drupal::service()`  to create a service inside of the block.